### ISSUE TITLE
Document Object ignore a field that is equal to None

### ISSUE THREAD

**Issue by aanagnostou** [IBM/cloudant-python-sdk#310]
**Describe the bug**
Constructing a Document Object with a field that is equal to None the field is ignored & not inserted in CouchDb database document.

**To Reproduce**

```Python
from ibmcloudant.cloudant_v1 import Document, CloudantV1, BulkDocs
import json

event_doc_1 = Document(
    id="0007241142412418284",
    type="event",
    userid="abc123",
    eventType="addedToBasket",
    productId="1000042",
    date="2019-01-28T10:44:22.000Z",
)
event_doc_2 = Document(
    id="0007241142412418285",
    type=None,
    userid="abc234",
    eventType="addedToBasket",
    productId="1000050",
    date="2019-01-25T20:00:00.000Z",
)
#
event_doc_3 = Document()
event_doc_3_json = r'{"_id":"0007241142412418285", "type": null, "userid":"abc234", "eventType":"addedToBasket", "productId":"1000050", "date":"2019-01-25T20:00:00.000Z"}'
#
bulk_docs = BulkDocs(docs=[event_doc_1, event_doc_2])
#
print("#============# bulk_docs #============#")
print(bulk_docs)
print("#============# Document.todict() #============#")
doc_2_json = json.dumps(event_doc_2.to_dict())
print(doc_2_json)
print("#============# Document Properties #============#")
print(event_doc_2.get_properties())
print("#============# Document.from_dict(_dict: Dict)  #============#")
print(event_doc_3.from_dict(json.loads(event_doc_3_json)))
print("#============# Dict from Json #============#")
print(json.loads(event_doc_3_json))
```

**Expected behavior**

docs is a list with <class 'ibmcloudant.cloudant_v1.Document'>

```
#============# bulk_docs #============#
{
  "docs": [
    {
      "_id": "0007241142412418284",
      "type": "event",
      "userid": "abc123",
      "eventType": "addedToBasket",
      "productId": "1000042",
      "date": "2019-01-28T10:44:22.000Z"
    },
    {
      "_id": "0007241142412418285",
      "type": None,
      "userid": "abc234",
      "eventType": "addedToBasket",
      "productId": "1000050",
      "date": "2019-01-25T20:00:00.000Z"
    }
  ]
}
```

**Screenshots**
```shell
#============# bulk_docs #============#
{
  "docs": [
    {
      "_id": "0007241142412418284",
      "type": "event",
      "userid": "abc123",
      "eventType": "addedToBasket",
      "productId": "1000042",
      "date": "2019-01-28T10:44:22.000Z"
    },
    {
      "_id": "0007241142412418285",
      "userid": "abc234",
      "eventType": "addedToBasket",
      "productId": "1000050",
      "date": "2019-01-25T20:00:00.000Z"
    }
  ]
}
#============# Document.todict() #============#
{"_id": "0007241142412418285", "userid": "abc234", "eventType": "addedToBasket", "productId": "1000050", "date": "2019-01-25T20:00:00.000Z"}
#============# Document Properties #============#
{'type': None, 'userid': 'abc234', 'eventType': 'addedToBasket', 'productId': '1000050', 'date': '2019-01-25T20:00:00.000Z'}
#============# Document.from_dict(_dict: Dict)  #============#
{
  "_id": "0007241142412418285",
  "userid": "abc234",
  "eventType": "addedToBasket",
  "productId": "1000050",
  "date": "2019-01-25T20:00:00.000Z"
}
#============# Dict from Json #============#
{'_id': '0007241142412418285', 'type': None, 'userid': 'abc234', 'eventType': 'addedToBasket', 'productId': '1000050', 'date': '2019-01-25T20:00:00.000Z'}
```

**Must gather (please complete the following information):**
 - IBM Cloud Sdk Core Version [3.15.3]  
 - SDK Version [0.1.4]
 - Python Version [Python 3.8.10]

**Comment by emlaver** [comment]
Hi @aanagnostou thanks for bringing this to our attention.  As a workaround while we look into this, you can use `BinaryIO`:
```python
event_doc_2_json = {"docs": [{'_id': '000724114241241828', 'type': None, ...}]}
event_doc_2_bytes = json.dumps(event_doc_2_json).encode('utf-8')

result = service.post_bulk_docs('test', bulk_docs=event_doc_2_bytes)
...
```

**Comment by ricellis** [comment]
This should be fixed by https://github.com/IBM/cloudant-python-sdk/pull/328 and available in the [0.2.1 release](https://github.com/IBM/cloudant-python-sdk/releases/tag/v0.2.1).
